src/main/java/programminglife/gui/controller/GuiCreateBookmarkController.java

```diff
@@ -48,7 +48,7 @@ private void initButtons() {
                     Alerts.warning("Center node is not present in graph").show();
                 } else if (inputRadius < 0) {
                     Alerts.warning("Radius can only contain positive integers").show();
-                } else if (txtBookmarkName.getLength() <= 0) {
+                } else if (!txtBookmarkName.getText().matches(".*\\w.*")) {
```

Comment:
Accepting any non-whitespace character would also be faster to match probably (linear instead of quadratic)?